frontend/src/components/Header/HeaderTitle/HeaderTitle.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import logo from './careandshare.png';
+
+import ('../style.css');
+
+class Title extends Component {
```

Comment:
i think u didn't need use class here, u can use stateless function

Reply:
@ahmadmshatat I edited it
please review it again